spec/index.html

```diff
@@ -10369,7 +10369,151 @@ <h3>Evaluation Semantics</h3>
 eval(D(G), Slice(L, start, length)) = Slice(eval(D(G), L), start, length)
             </pre>
           </div>
+
+          <!-- ValuesInsertion -->
+          <section>
+            <h3>Values Insertion and `EXISTS`</h3>
+            <p>The following sections contain draft of material for a revised
+              "`exists`" operation.
+            </p>
```

Comment:
```suggestion
            <p>This section contains draft material for a revised
              "`exists`" operation.
            </p>
```

Reply:
This is a DRAFT pull request.

These are process notes. They will be removed.